src/scripts/actions/UserActions.js

```diff
@@ -0,0 +1,30 @@
+/* @flow */
+
+import { createAction } from 'redux-actions'
+import Promise from 'bluebird'
+import Cookies from 'js-cookie'
+
+import { LOGIN_REQUESTED, LOGIN_SUCCESS, LOGOUT } from 'constants/ActionConstants'
+
+import type { Dispatch } from 'redux'
+
+const loginRequested = createAction(LOGIN_REQUESTED)
+const loginSuccess = createAction(LOGIN_SUCCESS)
+const logoutAction = createAction(LOGOUT)
+
+const sleep = (ms) => {
+  return new Promise(resolve => setTimeout(resolve, ms))
+}
+
+export const login = (email: string, password: string) => (dispatch: Dispatch<*>) => {
+  dispatch(loginRequested())
+  sleep(200).then(() => {
```

Comment:
What is the use of the `sleep`? Is this not actually logging in right now and just faking it? In the future would this be replaced by a call to `paratii-lib`?

Reply:
It's just to fake the time response of the lib call. In the future we replace it for the call

Reply:
Got it